grafana-dashboards/sre-capability-aus-ops.configmap.yaml

```diff
@@ -336,9 +997,9 @@ data:
       },
       "timepicker": {},
       "timezone": "",
-      "title": "AUS Ops Dashboard",
-      "uid": "bqA9EFwVk",
-      "version": 11,
+      "title": "AUS Cluster Upgrade Overview",
```

Comment:
I wonder if you updated the good file there 🤔

Reply:
no i did not :) thank you 🙇